dataflow/src/node/process.rs

```diff
@@ -53,7 +54,10 @@ impl Node {
                         // Send write-ACKs to all the clients with updates that made
                         // it into this merged packet:
                         if let Some(ex) = executor {
-                            senders.drain(..).for_each(|src| ex.send_back(src, ()));
+                            // TODO(ekmartin): work around this clone
```

Comment:
Not sure what a good approach would be here. I briefly tried making `send_back` take a reference instead, but not sure how feasible that is considering that it has to be stored in `Sendback`'s list of IDs.